web/client/epics/config.js

```diff
@@ -160,12 +161,16 @@ export const zoomToMaxExtentOnConfigureMap = action$ =>
         .delay(300) // without the delay the map zoom will not change
         .map(({config, zoomToExtent: extent}) => zoomToExtent(extent.bounds, extent.crs || get(config, 'map.projection')));
 
-export const calculateBboxOnConfigureMap = action$ =>
+export const calculateBboxOnConfigureMap = (action$, store) =>
     action$.ofType(MAP_CONFIG_LOADED)
-        .map(({config}) => {
-            const { center, zoom, size, mapStateSource, projection, viewerOptions, resolution } = (config?.map ?? {});
-            const bbox = getBbox(center, zoom);
-            return changeMapView(center, zoom, bbox, size, mapStateSource, projection, viewerOptions, resolution);
+        .switchMap(({config}) => {
+            const mapType = mapTypeSelector(store.getState());
+            if (mapType !== 'cesium') {
+                const { center, zoom, size, mapStateSource, projection, viewerOptions, resolution } = (config?.map ?? {});
+                const bbox = getBbox(center, zoom);
+                return Observable.of(changeMapView(center, zoom, bbox, size, mapStateSource, projection, viewerOptions, resolution));
+            }
+            return Observable.empty();
```

Comment:
I would avoid to add a new epic to solve the issue if was introduced of because it's adding a side effect affecting all the MAP_CONFIG_LOADED action, in addition is manipulating the state of the map by using changeMapView an action that usually is triggered by the map itself while panning or zooming.
This epic was introduced to solve this issue https://github.com/geosolutions-it/MapStore2/issues/7778 related to a missing update of chart if they are connected to the map viewport (bbox in the state). While reviewing the PR I noticed that once a user save the map the bbox in the state is undefined and this is causing the issue.
If we look the save workflow we noticed that we are replacing the current map content with the new one returned by the save response and this content is not including the bbox information

https://github.com/geosolutions-it/MapStore2/blob/ac760dfd4c4fc5a8702dcf218a9b1f87fd6bfa6d/web/client/epics/maps.js#L340

I would suggests following changes:

- [ ] remove this epic
- [ ] merge the saved map with the map in the state so we include also the current missing information

example

```js
function mergeMapState(state, data) {
    return {
        ...data,
        map: {
            ...mapSelector(state), // this contains the current bbox, missing in the response
            ...data?.map
        }
    };
}
```

then

```js
configureMap(mergeMapState(store.getState(), resource.data), rid)
```
I'm not sure if a similar merge map config exists in the code base, if so use the existing one

Note: I tested locally only with Save so please test that everything works also for SaveAs